src/contexts/TransactionContext.tsx

```diff
@@ -57,7 +57,9 @@ export function TransactionContextProvider(props: TransactionContextProviderProp
     payloadError: null,
     showBalance: false,
     formatFound: null,
-    estimatedFeeLoading: false
+    estimatedFeeLoading: false,
+    payloadHex: null,
```

Comment:
Sort all props please or at least move related stuff next to each other :)

Reply:
sure ... i will also do it on the `transactionTypes`